src/data/queries/pressRelease.ts

```diff
@@ -0,0 +1,59 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { NodePressRelease } from '@/types/drupal/node'
+import { PressRelease } from '@/types/formatted/pressRelease'
+import { RESOURCE_TYPES } from '@/lib/constants/resourceTypes'
+import { ExpandedStaticPropsContext } from '@/lib/drupal/staticProps'
+import {
+  entityBaseFields,
+  fetchSingleEntityOrPreview,
+} from '@/lib/drupal/query'
+import { get } from 'lodash'
+import { getNestedIncludes } from '@/lib/utils/queries'
+
+// Define the query params for fetching node--press_release.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude([
+    ...getNestedIncludes('field_press_release_downloads', ["media--image","media--document", "media--video"] ),
+    'field_press_release_contacts',
+    'field_press_release_pdf_version',
+    'field_press_release_office',
```

Comment:
`field_press_release_office` doesn't exist. 
`field_press_release_contacts` -> `field_press_release_contact`
also need to include the listing and administration fields (for lovell handling)

```js
  return new DrupalJsonApiParams().addInclude([
    ...getNestedIncludes('field_press_release_downloads', ["media--image","media--document","media--video"] ),
    'field_pdf_version.field_document',
    'field_press_release_contact',
    'field_listing',
    'field_administration'
     ])
```